test_data/rim_test3_fixed_output.txt

```diff
@@ -0,0 +1,120 @@
+║   │   │  │  │  │   │   │   │ 3 │   │   │   │   │   │       ║
+║   │   │  │  │  │   │   │   │ 4 │   │   │   │   │   │       ║
+║   │   │  └──┘6 │   │   │   │ 5 │   │   │   │   │   │       ║
+╚═══╧═══╧════════╧═══╧═══╧═══╧═══╧═══╧═══╧═══╧═══╧═══╧═══════╝
+╔═══╤═══╤═══╤════════╤═══╤═══╤═══╤═══╤═══╤═══╤═══╤═══╤═══════╗
+║   │   │   │  ┌──┐1 │   │   │ ▓ │ 2 │   │   │   │   │       ║
+║   │   │   │  │  │  │   │   │ 3 │   │   │   │   │   │       ║
+║   │   │   │  │  │  │   │   │ 4 │   │   │   │   │   │       ║
+║   │   │   │  └──┘6 │   │   │ 5 │   │   │   │   │   │       ║
+╚═══╧═══╧═══╧════════╧═══╧═══╧═══╧═══╧═══╧═══╧═══╧═══╧═══════╝
+╔═══╤═══╤═══╤═══╤════════╤═══╤═══╤═══╤═══╤═══╤═══╤═══╤═══════╗
+║   │   │   │   │  ┌──┐1 │   │ ▓ │ 2 │   │   │   │   │       ║
+║   │   │   │   │  │  │  │   │ 3 │   │   │   │   │   │       ║
+║   │   │   │   │  │  │  │   │ 4 │   │   │   │   │   │       ║
+║   │   │   │   │  └──┘6 │   │ 5 │   │   │   │   │   │       ║
+╚═══╧═══╧═══╧═══╧════════╧═══╧═══╧═══╧═══╧═══╧═══╧═══╧═══════╝
+╔═══╤═══╤═══╤═══╤═══╤════════╤═══╤═══╤═══╤═══╤═══╤═══╤═══════╗
+║   │   │   │   │   │  ┌──┐1 │ ▓ │ 2 │   │   │   │   │       ║
+║   │   │   │   │   │  │  │  │ 3 │   │   │   │   │   │       ║
+║   │   │   │   │   │  │  │  │ 4 │   │   │   │   │   │       ║
+║   │   │   │   │   │  └──┘6 │ 5 │   │   │   │   │   │       ║
+╚═══╧═══╧═══╧═══╧═══╧════════╧═══╧═══╧═══╧═══╧═══╧═══╧═══════╝
+╔═══╤═══╤═══╤═══╤═══╤═══╤════════╤═══╤═══╤═══╤═══╤═══╤═══════╗
+║   │   │   │   │   │   │  ┌──┐1 │ 2 │   │   │   │   │       ║
+║   │   │   │   │   │   │  │ 3│  │   │   │   │   │   │       ║
+║   │   │   │   │   │   │  │45│  │   │   │   │   │   │       ║
+║   │   │   │   │   │   │  └──┘6 │   │   │   │   │   │       ║
+╚═══╧═══╧═══╧═══╧═══╧═══╧════════╧═══╧═══╧═══╧═══╧═══╧═══════╝
+ ╔═══╤═══╤═══╤═══╤═══╤═══╤════════╤═══╤═══╤═══╤═══╤═══╤═══════╗ 
+┌──────────────────────────────────────────────────────────────┐
+│               The provided input is not valid!               │
+│                      Offending input: 0                      │
+└──────────────────────────────────────────────────────────────┘
+ ╚═══╧═══╧═══╧═══╧═══╧═══╧════════╧═══╧═══╧═══╧═══╧═══╧═══════╝ 
+╔═══╤═══╤═══╤═══╤═══╤═══╤════════╤═══╤═══╤═══╤═══╤═══╤═══════╗
+║   │   │   │   │   │   │  ┌──┐1 │   │ 2 │   │   │   │       ║
+║   │   │   │   │   │   │  │ 3│  │   │   │   │   │   │       ║
+║   │   │   │   │   │   │  │45│  │   │   │   │   │   │       ║
+║   │   │   │   │   │   │  └──┘6 │   │   │   │   │   │       ║
+╚═══╧═══╧═══╧═══╧═══╧═══╧════════╧═══╧═══╧═══╧═══╧═══╧═══════╝
+╔═══╤═══╤═══╤═══╤═══╤═══╤═══╤════════╤═══╤═══╤═══╤═══╤═══════╗
+║   │   │   │   │   │   │   │  ┌──┐1 │ 2 │   │   │   │       ║
+║   │   │   │   │   │   │   │  │ 3│  │   │   │   │   │       ║
+║   │   │   │   │   │   │   │  │45│  │   │   │   │   │       ║
+║   │   │   │   │   │   │   │  └──┘6 │   │   │   │   │       ║
+╚═══╧═══╧═══╧═══╧═══╧═══╧═══╧════════╧═══╧═══╧═══╧═══╧═══════╝
```

Comment:
This and the following lines of the output are wrong. The ▓ characters are missing from the central tile.